db/migrate/20200623165758_alter_type_field_on_school.rb

```diff
@@ -0,0 +1,5 @@
+class AlterTypeFieldOnSchool < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :schools, :type, :school_type
+  end
+end
```

Comment:
In this one PR, instead of doing migrations, you can simply edit the `..._create_schools.rb` file, drop your database and recreate it.